.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+# Python CircleCI 2.0 configuration file
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+version: 2
+jobs:
+  build:
+    docker:
+      # CircleCI maintains a library of pre-built images documented at
+      # https://circleci.com/docs/2.0/circleci-images/
+      # use `-browsers` prefix for selenium tests, e.g. `<image_name>-browsers`
+
+      # Python
+      - image: circleci/python:3.5.3
+        environment:
+          - TZ=America/New_York
+
+      # PostgreSQL
+      - image: circleci/postgres:9.6.2
+        environment:
+          - POSTGRES_USER=circleci
+          - POSTGRES_DB=fec-api-test
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - run:
+          name: install nodejs
+          command: |
+            sudo apt-get update && curl -sL https://deb.nodesource.com/setup_6.x | sudo bash - && sudo apt-get install -y nodejs
+
+      - restore_cache:
+          keys:
+          - fec-api-dependencies-{{ checksum "requirements.txt" }}-{{ checksum "requirements-dev.txt" }}-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - fec-api-dependencies-
+
+      - run:
+          name: install dependencies
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install bandit codecov
+            pip install -r requirements.txt -r requirements-dev.txt
+            npm install -g swagger-tools
```

Comment:
You'll want to `sudo` here for the global swagger install. Pretty sure this is the cause of your build failure, not caching.

Reply:
Ah okay, thank you!

Reply:
@LinuxBozo is there anything required when using `sudo` for this, like in Travis, or safe and okay to just put in?

Reply:
With the particular docker image you are using, from CircleCI, you just can use it out of the box. Otherwise, yes, you would need to make sure that your container supports using sudo, or run everything as root (usually a bad idea). 

BTW, here are basics and how Circle is doing it (and really same pattern can be used in any container):
 - makes sure sudo installed (https://github.com/circleci/circleci-images/blob/master/shared/images/Dockerfile-basic.template#L9..L13)
 - create a user https://github.com/circleci/circleci-images/blob/master/shared/images/Dockerfile-basic.template#L66..L67
 - give user sudo access https://github.com/circleci/circleci-images/blob/master/shared/images/Dockerfile-basic.template#L68..L69
 - drop privileges to that user to run the entrypoint/cmd
https://github.com/circleci/circleci-images/blob/master/shared/images/Dockerfile-basic.template#L74

Reply:
Awesome, thank you very much!